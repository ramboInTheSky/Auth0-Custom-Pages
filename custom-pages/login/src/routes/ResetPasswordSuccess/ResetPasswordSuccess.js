import React from 'react'
import Alert from '@nice-digital/nds-alert'
// import { Link } from 'react-router'

// local imports
import { hideNav } from '../../util'

// style
import './ResetPasswordSuccess.scss'

export const ResetPasswordSuccess = () => {
  // setTimeout(() => (document.location = '/login'), 5000)
  hideNav()
  return (
    <div>
      <h3>Thank you!</h3>
      <Alert type="success">
        Your password has been changed and you can now use it to sign in {' '}
        {/* <Link data-qa-sel="resetPasswordSuccess-link-to-login" to="/login">
          {' '}
          click here{' '}
        </Link> */}
      </Alert>
    </div>
  )
}

export default ResetPasswordSuccess
