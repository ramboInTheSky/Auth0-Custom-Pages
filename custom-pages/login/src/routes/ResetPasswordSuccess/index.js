import ResetPasswordSuccess from './ResetPasswordSuccess'

export default {
  path: 'resetsuccess',
  component: ResetPasswordSuccess
}
