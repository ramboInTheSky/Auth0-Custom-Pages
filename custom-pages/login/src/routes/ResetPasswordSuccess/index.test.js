import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-14'
import ResetPasswordSuccess from './'

Enzyme.configure({ adapter: new Adapter() })

describe('ResetPasswordSuccess components', () => {
  it('should render <ResetPasswordSuccess /> correctly', () => {
    expect(shallow(<ResetPasswordSuccess.component />)).toMatchSnapshot()
  })
})
