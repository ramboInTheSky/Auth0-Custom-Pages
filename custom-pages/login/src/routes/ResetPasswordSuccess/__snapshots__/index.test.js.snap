// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordSuccess components should render <ResetPasswordSuccess /> correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ResetPasswordSuccess />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Thank you!
        </h3>,
        <Alert
          type="success"
        >
          Your password has been changed and you can now use it to sign in 
           
        </Alert>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Thank you!",
        },
        "ref": null,
        "rendered": "Thank you!",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            "Your password has been changed and you can now use it to sign in ",
            " ",
          ],
          "type": "success",
        },
        "ref": null,
        "rendered": Array [
          "Your password has been changed and you can now use it to sign in ",
          " ",
        ],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Thank you!
          </h3>,
          <Alert
            type="success"
          >
            Your password has been changed and you can now use it to sign in 
             
          </Alert>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Thank you!",
          },
          "ref": null,
          "rendered": "Thank you!",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              "Your password has been changed and you can now use it to sign in ",
              " ",
            ],
            "type": "success",
          },
          "ref": null,
          "rendered": Array [
            "Your password has been changed and you can now use it to sign in ",
            " ",
          ],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFourteenAdapter {
      "options": Object {
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "prevContext": true,
          },
          "getChildContext": Object {
            "calledByRenderer": true,
          },
        },
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
